package.json

```
@@ -43,6 +43,8 @@
     "template-starter:start": "yarn --cwd application-templates/starter start",
     "template-starter:start:prod:local": "yarn --cwd application-templates/starter start:prod:local",
     "build": "NODE_ENV=production lerna run build --no-private",
+    "postbuild": "yarn check-links",
```

This should only be used as post build of the website.

```suggestion
    "postbuild:website": "yarn check-links",
```

---

Better to include it in the postbuild hook of the website package